src/components/Layout/Header.jsx

```diff
@@ -8,35 +8,43 @@ const Header = ({ setOpen }) => {
   return (
     <>
       <HeaderWrapper>
-        <Typography fontWeight="bold">MCY</Typography>
+        <LogoWrapper>
+          <Typography fontWeight="bold">MCY</Typography>
+        </LogoWrapper>
+        <MenuIconWrapper>
+          <IconButton onClick={() => setOpen(true)}>
```

Comment:
```suggestion
          <IconButton onClick={handleDrawerOpen}>
```
handleDrawerOpen 핸들러 만들어서 따로 분리해보도록 ! 
이 부분 관련해서 대현이 PR에도 리뷰 남겼는데 확인 부탁